### DISCUSSION TITLE
H4 on Warco WM180

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#159] [Show and tell]
Just thought id share my build and also id like to thank Maxim for this project, im getting an incredible finish on my cuts!

Here is the enclosure im using for the power supply and drivers. Its a cheap (14.99) metal letter box from amazon. There is still space inside for a raspberry pi which will be talking to the control board over serial.

![402B8AEE-9B73-492C-88F0-A0BFE553A01B](https://github.com/kachurovskiy/nanoels/assets/2408274/cf3fecb4-e672-4843-a4a5-51f6c7d7a0d2)

Here is my attachment for mounting the encoder. Due to space constraints I had to use gears to transfer the motion. It goes from 40 to 50 to 40 which means its a 1:1 ratio.
![1C9D9AFE-E122-4EB9-B03B-5EA1EAE7388C](https://github.com/kachurovskiy/nanoels/assets/2408274/fd5514de-eab9-4f0e-ae71-10481aff47e6)
![A6D03D2D-B48A-48FE-840F-7BB4E1C655B4](https://github.com/kachurovskiy/nanoels/assets/2408274/92905252-5973-4f18-b666-c5b9ca4c59e0)

Ive mounted the control box at the front using a 3d printed plate that bolts into existing bolt holes
![AA24800C-22C9-4C7F-B5FF-AB9EFA43EA50](https://github.com/kachurovskiy/nanoels/assets/2408274/652715ac-002e-4d53-b5c3-342359c75c3c)

The metal control box is bolted to my lathe stand
![4B6E4A8B-507C-4823-AB55-F19EEF43ECB9](https://github.com/kachurovskiy/nanoels/assets/2408274/e7ebbc36-b567-42f3-a2ee-4d1717e407ee)

Still some cable management to do
![F042C73C-2C24-4433-8FAF-6720703CFEF7](https://github.com/kachurovskiy/nanoels/assets/2408274/cebff504-0a63-44fb-a303-1df88af9e73d)

Im very happy with how its turned out. I got the backlash on the Z down to 0.02mm by adjusting my half nut.
I intend to make a solid tool post and yeet my compound slide.
![854A7723-03C0-4485-B907-686DF865798B](https://github.com/kachurovskiy/nanoels/assets/2408274/7db9a9c8-0e41-4468-b902-a89d5918ead6)

Ill upload my designs to thingivers soon and add a link here, i need go make a few more tweaks.

**Comment by kachurovskiy** [comment]
Awesome build! That grease is ready to become airborne 🚀 😄

**Reply by Festivejelly** [reply]
Haha thats why I swiftly replaced the cover. I think what ill end up doing is printing the gears in nylon but for now I think they will do.

**Comment by kachurovskiy** [comment]
@Festivejelly can you please share 3D print designs? Colin in https://github.com/kachurovskiy/nanoels/issues/166 asks for help with encoder mounting on WM180.

**Comment by Festivejelly** [comment]
This is my solution 
![image](https://github.com/kachurovskiy/nanoels/assets/2408274/3e257554-c027-4df7-8fc9-b79da426bd94)


Colins WM180 looks likely different to mine, I dont see any bolt holes on the main plate but He could quite easily just drill some through both the green plate plus the 3d printed plate.

There are 2 3D printed gears. One 50 tooth which meshes with the spindle and one 40 tooth which is on the encoder.

![image](https://github.com/kachurovskiy/nanoels/assets/2408274/fab0a7aa-c6d1-4a98-9173-e6317529a846)

The spindle gear is supported by 2 bearings which are 16mm OD 8mm ID.

A small washer sits under the outer bearing

![image](https://github.com/kachurovskiy/nanoels/assets/2408274/a8598a2d-a7f1-4be1-9184-115fa4b0e788)

The back plate has 5 holes which allows the top plate to screw into it. I used M3 x 25 and they self tap into the back plate.

![image](https://github.com/kachurovskiy/nanoels/assets/2408274/9ccbe814-cc0c-4376-b6f7-a47a5329b46e)

You will need to figure out a way of mounting the back plate. I tried tape but it just didnt hold. But I had 2 threaded holes in the back so I simply just drilled through the plastic back plate and then fastened it via the holes.

If you dont have these holes on your lathe you might need to drill and tap some, but be careful.

I also remixed the cross slide mount as the distance between bolt centres was different on mine:

https://www.thingiverse.com/thing:6311146

And here is the link to the encoder mount. Its not perfect but itle do in a pinch. Seems to be working fine for me. You might need to scale it based on how accurate your 3d printer is.

https://www.thingiverse.com/thing:6418276

**Reply by Colinsp** [reply]
Thanks very much I will print those up tomorrow

**Reply by EricB420** [reply]
Clean and nice solution for mounting the encoder ! What plastic do you recommend for the gears ?

**Reply by Festivejelly** [reply]
> Clean and nice solution for mounting the encoder ! What plastic do you recommend for the gears ?

Sorry for late reply on this. I use PC-ABS.
